tests/conftest.py

```diff
@@ -848,9 +880,10 @@ def __init__(self, server='Xephyr', screens=[(800, 600)], extra_args=[]):
         # ---------------------------
         self.screen_rects = []
         current_x_offset = 0
+        depth = 24
         for idx, (width, height) in enumerate(screens):
             self.screen_rects.append([current_x_offset, 0, width, height])
-            geo = '{}x{}x8'.format(width, height)
+            geo = '{}x{}x{}'.format(width, height, depth)
```

Comment:
Would an f-string make it more readable?

Reply:
I think it's fine this way because `'{}x{}x{}'` is very short.

Reply:
True, it's just my personal taste, but both versions are fine, yes!